### ISSUE TITLE
Doesn't handle class declared inside a method

### ISSUE THREAD

**Issue by Graham42** [thorbenvh8/prettier-java#48]
Sample code:
```java
public class MyTest {
    public void testMethod() throws Exception {
        class MyLocalThing {
        }
    }
}
```

Output:
```
Unknown Java node: { type: 'LOCAL_TYPE_DECLARATION',
  modifiers: [],
  declaration:
   { type: 'CLASS_DECLARATION',
     name: { type: 'IDENTIFIER', value: 'MyLocalThing' },
     typeParameters: undefined,
     extends: undefined,
     implements: undefined,
     body: { type: 'CLASS_BODY', declarations: [] } } }
public class MyTest {

  public void testMethod() throws Exception {
  }

}
```

P.S. I know this is probably strange. :wink:  But I'm trying to run `prettier-java` against a large existing Java code base.

**Comment by thorbenvh8** [comment]
fixed in next release

**Comment by thorbenvh8** [comment]
Fixed in 0.0.10!
Thanks for the raising the issue! 👍
